### PR TITLE
Block components 01 02

### PR DESCRIPTION
**Pull request by wilson1000-MoJ**
- adds block components 01 + 02
- localises build files (css/js) in  build directory
- ignores build directory

### CODE REVIEW

File: style-gutenburg.scss [hunk 1]
```diff
@@ -3,3 +3,5 @@
    color: red;
  }
 }
+
+@import 'src/**/*.scss';
```
Comment (RobLoweMOJ) [inline style-gutenburg.scss:7]: This is already in this file not sure why its not picking it up. On the compare.

File: .gitignore [hunk 2]
```diff
@@ -4,4 +4,5 @@ node_modules
 .DS_Store
 vendor/*
 mojblocks.zip
+build/*
```
Comment (RobLoweMOJ) [inline .gitignore:7]: This wasalso added by adam and is in master.. not sure why the compare isnt working.

File: src/01-highlights-list/index.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                listTitle,
+                listText
+            }
+        } = props;
+
+        return (
+            <div className="mojuk-grid-column-width mojuk-highlights-list--type">
+                <div className="mojuk-highlights-list__heading-container">
+                    <h3 className="mojuk-highlights-list__heading">
+				 <span role="text">
```
Comment (BNewing) [inline src/01-highlights-list/index.js:93]: Ooh - what's the rationale behind `role="text"`? I've not seen that one before so I had a bit of a read up, and it looks like it's not in the [official ARIA spec at present](https://www.w3.org/WAI/PF/aria/roles#textbox). It looks like the [Working Group have considered adding it in once or twice](https://github.com/w3c/aria/issues/870), but it seems to be still a bit of a debate. It's an interesting one!

I'd say we maybe don't need it here as I'm assuming the RichText component wrappers everything in the right semantic HTML, and [the first rule of ARIA is don't use it](https://www.w3.org/TR/using-aria/#firstrule) - but that's an assumption on my part! What are your thoughts?

Comment (wilson1000-MoJ) [reply src/01-highlights-list/index.js:93]: This is up for debate, I took the role from the NHS plugin. My feeling is we build the components and consider accessibility during build but then discuss accessibly in a dev meet once all 7 have been created to address any concerns. Does that sound good?

File: src/01-highlights-list/index.js [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+    save: props => {
+        let {
+            attributes: {
+                listTitle,
+                listText
+            }
+        } = props;
+
+        return (
+            <div className="mojuk-grid-column-width mojuk-highlights-list--type">
```
Comment (RobLoweMOJ) [inline src/01-highlights-list/index.js:90]: @wilson1000-MoJ Just seen this mojuk-grid-column-width class. Just curious where is this defined in sass? Or is this just our own naming convention?

**Pull request merged**